### DISCUSSION TITLE
Shuffle list

### DISCUSSION THREAD

**Discussion by sugar-king** [t1m0thyj/WinDynamicDesktop#311] [Q&A] [answered]
I was wondering how exactly does the shuffle work?

Does it shuffle all available (downloaded) themes or just those in the shuffle history?

**Comment by t1m0thyj** [comment, marked as answer]
The shuffle list is all downloaded themes. These themes are displayed in a random order, a new one each day. If there are N themes, after N days the cycle will start over again, in a new random order.
